emu/src/arm/arm7tdmi.rs

```diff
@@ -183,6 +197,97 @@ impl Arm7tdmi {
         !(load_store && rd_source_destination_register == 0xF)
     }
 
+    /// Stores the banked registers of the current mode to the register bank.
+    /// It should be used before leaving the current mode.
+    pub fn bank_registers(&mut self) {
```

Comment:
Is it a better name `store_bank_registers`?

Reply:
What do you think about `store_registers_in_bank`?

Reply:
Is good, just use a verb with an action for what we mean :D

Reply:
Done thanks!